jupyter_client/channels.py

```diff
@@ -121,7 +121,7 @@ def _poll(self, start_time: float) -> t.List[t.Any]:
         events = []
         while True:
             try:
-                events = self.poller.poll(int(1000 * until_dead))
+                events = self.poller.poll(1)
```

Comment:
Does it mean we are going to iterate every milllisecond?

Reply:
Well a bit longer than that because of the overhead, the time could be adjusted. How does 100ms sounds?

Reply:
On second thought, I don't think the loop is necessary at all. I removed it.